ofrak_core/test_ofrak/components/test_gzip_component.py

```diff
@@ -1,87 +1,55 @@
-import os
-import subprocess
-import tempfile
 from gzip import GzipFile
 from io import BytesIO
+from pathlib import Path
 
 import pytest
 
-from ofrak import OFRAKContext
 from ofrak.resource import Resource
 from ofrak.core.gzip import GzipData
 from pytest_ofrak.patterns.compressed_filesystem_unpack_modify_pack import (
     CompressedFileUnpackModifyPackPattern,
 )
-from pytest_ofrak.patterns.unpack_modify_pack import UnpackModifyPackPattern
 
 
-class TestGzipUnpackModifyPack(CompressedFileUnpackModifyPackPattern):
+class GzipUnpackModifyPackPattern(CompressedFileUnpackModifyPackPattern):
+    """Template for tests that test different inputs the gzip component should support
+    unpacking."""
```

Comment:
```suggestion
    """
    Template for tests that test different inputs the gzip component should support
    unpacking.
    """
```

Minor, but FYI for future that we format docstrings in a specific way. [More info in the OFRAK contributor style guide docs.](https://ofrak.com/docs/contributor-guide/getting-started.html#functions-and-methods)